xml_inscriptions/metadata_only/NY.NY.MMA.G.38.11.13.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?oxygen RNGSchema="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" type="xml"?><TEI xmlns:xi="http://www.w3.org/2001/XInclude" xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader><!-- testing1 -->
+      <fileDesc>
+         <titleStmt>
+            <title>NY.NY.MMA.G.38.11.13</title>
+         </titleStmt>
+         <publicationStmt>
+            <xi:include  href="http://library.brown.edu/usep_data/resources/include_publicationStmt.xml">
+               <xi:fallback>
+                  <p>The Publications Statement should appear here. (c) Brown University</p>
+               </xi:fallback>
+            </xi:include>
+            <idno type="USEpigraphy" xml:id="NY.NY.MMA.G.38.11.13">NY.NY.MMA.G.38.11.13</idno>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc>
+               <msIdentifier>
+                  <region>NY</region>
+                  <settlement>NY</settlement>
+                  <institution>MMA</institution>
+                  <repository/>
+                  <idno>38.11.13</idno>
+               </msIdentifier>
+               <msContents>
+                  <textLang mainLang="grc" otherLangs=""/>
+                  <msItem>
+                     <p/>
+                  </msItem>
+               </msContents>
+               <!--  *************************************  <physDesc>  ********************************* -->
+               <physDesc>
+                  
+                  <!--  *************************************  <objectDesc>  *********************************  -->
+                  
+                  <objectDesc ana="#stele">
+                     <supportDesc ana="#stone">
+                        <support>
+                           <p> Fragment of marble stele</p>
```

Comment:
Don't put spaces after your open-elements; the computer will pick up that there is white-space there.